kitsune/search/v2/es7_utils.py

```diff
@@ -92,8 +92,8 @@ def index_object(doc_type_name, obj_id):
     model = doc_type.get_model()
 
     obj = model.objects.get(pk=obj_id)
-    doc = doc_type.prepare(obj)
-    doc.save()
+    payload = doc_type.prepare(obj)
+    doc_type(_id=obj_id).update(payload, doc_as_upsert=True)
```

Comment:
I don't think we want every kind of doc to be sent as an update, I imagine it impacts on performance, and could lead to data persisting in ES if - for example - a user sets their display name to an empty value.

We could add a class attribute here like `merge_docs` and use that here as well as the base `prepare` method - this would also avoid having to add a `prepare` method on the wiki doc.

Reply:
This would lead to some docs using the update option and others the create one. I feel that whether is an update or not it's not part of the Document. The document needs to go into the index but it should not decide if it's an update or a create operation. I will try to update the code here to create a document if it's not present or update if exists. 

Regarding data persistency, I am not sure I understand the comment